lib/ash_json_api/json_schema/open_api.ex

```diff
@@ -1519,6 +1526,8 @@ if Code.ensure_loaded?(OpenApiSpex) do
         paths_list
         |> Enum.group_by(fn {path, _route_op} -> path end, fn {_path, route_op} -> route_op end)
         |> Map.new(fn {path, route_ops} -> {path, struct!(PathItem, route_ops)} end)
+        |> Enum.sort_by(&elem(&1, 0))
+        |> Enum.into(%{})
```

Comment:
By putting them back into a map we un-sort them again. The tests likely pass because of having less than 32 keys or something loke that. Perhaps open api spex has some tooling around this?